sdk/template/template/tsconfig.json

```diff
@@ -1,28 +1,28 @@
 {
```

Comment:
we should extend the package.tsconfig.json in the root instead if possible

Reply:
I was going to do that for the remaining packages in another PR but I can take care of the template here and it will probably be easier.